docs/releases/5.0.md

```diff
@@ -141,6 +141,40 @@ If there are custom styles in place for the `ModelAdmin`'s header content or mor
 | Heading & search (contains `h1`) | `.left.header-left`   | `.left`         |
 | Action buttons (`header_extra`)  | `.right.header-right` | `.right`        |
 
+### Tag fields via Tagit
+
+The `AdminTagWidget` widget has now been migrated to a Stimulus controller, if using this widget in Python, no changes are needed to adopt the new approach.
+
+If the widget is being instantiated in JavaScript or HTML with the global util `window.initTagField`, this undocumented util should be replaced with the new
+`data` attributes only approach.
+
+The global util will be removed in a future release, additionally any direct usage of the jQuery widget in JavaScript (e.g. `$('#my-element).tagit()`) should be removed.
+
+#### Old syntax
+
+```html
+<input id="id_tags" type="text" value="popular, technology" hidden />
```

Comment:
Nitpicky small comment: I would find this easier to compare if there were line breaks like in the example for the new syntax

Reply:
Good call. I originally had the others all in one line but after I ran prettier it wrapped the below attributes. 

I'll make the old syntax match this also.

Reply:
This is done. Thanks @nmorduch